Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -76,16 +124,52 @@ private AtlassianUser ToAtlassianUser(JiraUser user, long instanceId)
             };
         }
 
-        private void UpdateDbUser(AtlassianUser dbUser, AtlassianUser updatedUser)
+        private List<AtlassianWorklog> GetWorklogsFromIssueWorklogs(List<JiraIssue> issueWorklog, List<AtlassianUser> users, long instanceId)
+        {
+            var worklogs = new List<AtlassianWorklog>();
+
+            foreach (var issue in issueWorklog)
+            {
+                foreach (var worklog in issue.fields.worklog.worklogs)
```

Comment:
can issue.fields.worklog be null?

Reply:
It can't be null, since the search is explicitly with WorklogAuthors and WorklogDate.